docs/xls-65d-single-asset-vault/concepts/single-asset-vault.md

```diff
@@ -34,9 +34,15 @@ Depositors can deposit assets to receive shares, which represent their proportio
 
 Since the XRP Ledger is an account-based blockchain, all assets must be held by an account. A `Vault` ledger entry cannot hold assets directly, so a [pseudo-account](pseudo-account.md) is created to hold assets on its behalf. This stand-alone account cannot receive funds or send transactions, and exists solely to store assets and issue shares.
 
-Each share is represented on-chain as an MPT, issued by the vault's pseudo-account.
+Each share is represented on-chain as an MPT, issued by the vault's pseudo-account. Since MPTs can only exist as whole number units, the vault uses a `Scale` setting to convert fractional asset amounts into whole number shares.
 
-Depending on the connected protocol, vault shares may be yield-bearing, meaning shareholders could redeem shares for more or less liquidity than they originally deposited. This is because the total asset balance in the vault can grow or shrink over time, affecting the value of each share. However, the vault asset (e.g., USDC or XRP) does not generate yield on its own.
+The scale behavior varies based on the type of asset held by the vault:
+
+- **XRP**: Uses a fixed scale that aligns with XRP's native structure, where one share represents one drop.
+- **Trust Line Token**: Allows configurable precision (default preserves 6 decimal places).
+- **MPT**: Uses a 1-to-1 relationship between MPT units and shares.
+
+Depending on the connected protocol, vault shares may be yield-bearing, meaning shareholders could redeem shares for more or less liquidity than they originally deposited. This is because the total asset balance in the vault can grow or shrink over time, affecting the value of each share. However, the vault asset (e.g., USDC, XRP) does not generate yield on its own.
```

Comment:
```suggestion
Depending on the connected protocol, vault shares may be yield-bearing, meaning shareholders could redeem shares for more or less liquidity than they originally deposited. This is because the total asset balance in the vault can grow or shrink over time, depending on connected protocols such as lending, which affects the value of each share. However, the vault asset (e.g., USDC, XRP) does not generate yield on its own.
```